Detectors/TOF/workflow/src/CompressedDecodingTask.cxx

```diff
@@ -141,8 +143,12 @@ void CompressedDecodingTask::run(ProcessingContext& pc)
 {
   mTimer.Start(false);
 
-  o2::base::GRPGeomHelper::instance().checkUpdates(pc);
-  const uint32_t norbits = o2::base::GRPGeomHelper::instance().getGRPECS()->getNHBFPerTF();
+  int norbits = mNorbitsPerTF;
+  if (norbits == -1) {
+    o2::base::GRPGeomHelper::instance().checkUpdates(pc);
+    norbits = o2::base::GRPGeomHelper::instance().getGRPECS()->getNHBFPerTF();
+    mNorbitsPerTF = norbits;
+  }
   Utils::setNOrbitInTF(norbits);
   mDecoder.setNOrbitInTF(norbits);
```

Comment:
these two `setNOrbitInTF` methods do the same and the number also needs to be set once in a run. Could you fix this as well?